package/boot/uboot-layerscape-armv8_32b/Makefile

```diff
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=uboot-layerscape-armv8_32b
-PKG_SOURCE_DATE:=2018-01-22
+PKG_VERSION:=lsdk-1806
```

Comment:
Yes. Since the packages were from LSDK release,  I'd like to use the release tag as the PKG_VERSION.
When upgrade these packages, we update them to the commit which has the release tag.